README.md

```diff
@@ -380,7 +402,7 @@ method takes two arguments:
 ### Profunctor
 
 A value that implements the Profunctor specification must also implement
-the Functor specification.
+the Functor and Contravariant Functor specifications.
```

Comment:
Thanks, @joneshf.

I think I am also pro removing it as a dependency. Are you happy for me to push up the removal, @scott-christopher?